setup.cfg

```diff
@@ -14,3 +14,7 @@ ignore_missing_imports = True
 
 [mypy-scipy.spatial.*]
 ignore_missing_imports = True
+
+[mypy-scipy.spatial.transform.rotation]
+ignore_missing_imports = True
+
```

Comment:
Was this needed to make mypy happy? (I would believe the one above would allow all under scipy.spatial

Reply:
Good catch! I think I missed it in a rebase :D It has been removed now!